### PR TITLE
Add realtime api client

### PR DESCRIPTION
**Pull request by viveknimkarde**
- Add realtime emotion api
- Add realtime speaker identification api
- Add apikey validation from kong
- Add sample codes for realtime api 
- Fix python version test

### CODE REVIEW

File: deepaffects/realtime/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# from realtime_api_util import get_segment, encode_to_base64, audio_segment_to_base64, make_input_audio_segments, get_deepaffects_client
```
Comment (sushant-hiray) [inline deepaffects/realtime/__init__.py]: If you don't need this line, remove it

File: deepaffects/realtime/deepaffects_realtime_pb2_grpc.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        Accepts a stream of SegmentChunk sent while a route is being traversed,
+        while receiving other SegmentSpeaker (e.g. from other users).
+        """
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+        context.set_details('Method not implemented!')
+        raise NotImplementedError('Method not implemented!')
+
+    def IdentifyEmotion(self, request_iterator, context):
+        # missing associated documentation comment in .proto file
+        pass
```
Comment (sushant-hiray) [inline deepaffects/realtime/deepaffects_realtime_pb2_grpc.py]: This `pass` shouldn't be needed

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,7 @@
 # ref: https://docs.travis-ci.com/user/languages/python
 language: python
 python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
```
Comment (sushant-hiray) [inline .travis.yml]: Why are these versions removed?

File: deepaffects/realtime/deepaffects_realtime_pb2_grpc.py (hunk 2 above)
Comment (viveknimkarde) [reply deepaffects/realtime/deepaffects_realtime_pb2_grpc.py]: its auto generated file.
`# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!`
we have implemented this function in server.py

File: .travis.yml (hunk 3 above)
Comment (viveknimkarde) [reply .travis.yml]: travis was installing python 3.3
and minimum version test was failing even before testing any python code

Comment (sushant-hiray) [reply .travis.yml]: just remove `3.3`, keep other versions.

**Pull request merged**